AdminServer/appscale/admin/__init__.py

```diff
@@ -93,6 +93,20 @@ def wait_for_port_to_open(http_port, operation_id, deadline):
 
     yield gen.sleep(1)
 
+  for load_balancer in appscale_info.get_load_balancer_ips():
```

Comment:
should you just skip the login_ip since you tested it just before getting into this loop?

Reply:
That would be more efficient, but I'm not sure how to find out which load balancer private IP matches the public login IP.